src/redis-benchmark.c

```diff
@@ -1452,6 +1452,20 @@ int parseOptions(int argc, char **argv) {
         } else if (!strcmp(argv[i],"-a") ) {
             if (lastarg) goto invalid;
             config.conn_info.auth = sdsnew(argv[++i]);
+        } else if (!strcmp(argv[i], "--readpass")) {
+            FILE *fp;
+            sds filepass = sdsempty();
+            char buf[32];
+            if ((fp = fopen(argv[++i], "r")) == NULL) {
+                fprintf(stderr, "can't open the file '%s'\n", argv[i]);
```

Comment:
Maybe include strerror(errno) in the output?

```suggestion
                fprintf(stderr, "Can't open the file '%s': %s\n", argv[i],
                        strerror(errno));
```